### ISSUE TITLE
How to do entropy regularization?

### ISSUE THREAD

**Issue by asdfqwer2015** [modelscope/ms-swift#7331]
Hi:
I'm currently using ms-swift for GRPO (Group Relative Policy Optimization‌) training and would like to add entropy regularization to improve exploration and prevent premature convergence. However, I couldn't find clear documentation or examples on how to enable this feature, particularly when launching training via scripts.
Could you please provide guidance or implement support for entropy regularization in GRPO training?
Thanks.

**Comment by hjh0119** [comment]
Currently, the entropy-related algorithm only supports [entropy_mask](https://swift.readthedocs.io/en/latest/Instruction/GRPO/AdvancedResearch/entropy_mask.html).

You can modify the code in GRPOTrainer.

**Comment by asdfqwer2015** [comment]
Thanks for the clarification. Regarding entropy_mask: since it relies on entropy statistics that require larger batch sizes for accurate estimation, this inevitably slows down convergence. In RLVR, unlike supervised learning, there doesn't seem to be theoretical support for compensating with a proportionally increased learning rate. This contrasts with direct entropy regularization, which doesn't necessitate larger batches. Any suggestions to address this trade-off?
